### PR TITLE
Add logical backup for azure

### PR DESCRIPTION
**Pull request by stgrace**
This PR adds support for logical backups to be pushed to Azure

In the backup script used to create a logical backup and push to cloud, I had to temporarily write the logical backup to a file in tmp, because Azure's cli cannot by upload data from stdin to a file in a storage account.
There is an extension that can do so, but this extension only works with strings, and fails when you pipe the result of the compress to it.  (https://github.com/Azure/azure-cli-extensions/blob/main/src/storage-blob-preview/azext_storage_blob_preview/_help.py#L172)

### CODE REVIEW

File: charts/postgres-operator/crds/operatorconfigurations.yaml [hunk 1]
```diff
@@ -489,6 +489,12 @@ spec:
                     type: string
                     pattern: '^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$'
                     default: "30 00 * * *"
+                  logical_backup_storage_account_name:
```
[the remark this answers is not in the document]
Comment (stgrace) [reply charts/postgres-operator/crds/operatorconfigurations.yaml]: Added a prefix 👍

**Comment by rgarcia89** [comment]
I would love to see this merged

**Comment by FxKu** [comment]
:+1:

**Comment by idanovinda** [comment]
👍

**Pull request merged**